src/render/vita/SDL_render_vita.c

```diff
@@ -47,30 +47,46 @@ extern int SDL_RecreateWindow(SDL_Window *window, Uint32 flags);
 
 static SDL_Renderer *VITA_CreateRenderer(SDL_Window *window, Uint32 flags);
 static void VITA_WindowEvent(SDL_Renderer *renderer, const SDL_WindowEvent *event);
+static SDL_bool VITA_SupportsBlendMode(SDL_Renderer * renderer, SDL_BlendMode blendMode);
```

Comment:
This file is the one that has changed the most. Basically because now render drivers work with queues, so commands to draw/clear/whatever are queued, and then the queue is processed.

In the end I haven't done changes to the actual render code: I just added code to queue those commands, and then invoke the existing render code when those commands are actually processed in the queue.